getItems.php

```diff
@@ -30,7 +30,12 @@ function getAllItems() {
                     <div class='itemBoxButtons'>
                     <button class='track-btn'>+ Track</button>";
             if (isset($_SESSION["loggedin"]) && $_SESSION["loggedin"] == TRUE) {
-                echo "<button class='details-btn'>View Details</button>";
+                $_SESSION["ITEMNAME"] = $row['itemName'];
+                $_SESSION["PRICE"] = $row['price'];
+                $_SESSION["ITEMID"] = $row['product_id'];
+                $_SESSION["IMG"] = $imgsrc;
+        
+                echo "<button class='details-btn'><a href=\"ViewDetails.php\">View Details</a></button>";
```

Comment:
```suggestion
                echo "<button class='details-btn'><a href='ViewDetails.php?productId=".$row['product_id']."'>View Details</a></button>";
```

Reply:
And then use the id to get the details for the product on the DB in ViewDetails.php